x/pki/types/message_add_pki_revocation_distribution_point.go

```diff
@@ -108,6 +120,62 @@ func (msg *MsgAddPkiRevocationDistributionPoint) verifyPAI(cert *x509.Certificat
 		return pkitypes.NewErrCRLSignerCertificatePidNotEqualMsgPid(pid, msg.Pid)
 	}
 
+	if msg.CrlSignerDelegator != "" {
+		if cert.AuthorityKeyID == "" {
```

Comment:
There should probably be a format check, see [`VerifyRedictionPointIssuerSubjectKeyIDFormat`](https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/7f9ce724fb1ff2394c2241c5dd3e62608a0cf49a/x/pki/types/types.go#L9)

Can you please also cover this case in unit tests?